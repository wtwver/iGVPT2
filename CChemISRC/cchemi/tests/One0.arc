

                     SUMMARY OF  PM3 CALCULATION, Site No: 1079

                                                       MOPAC2016 (Version: 19.338L)
                                                       Wed Mar  4 14:24:09 2020

           Empirical Formula: H2 O  =     3 atoms

 PM3 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =        -53.43270 KCAL/MOL =    -223.56244 KJ/MOL
          TOTAL ENERGY            =       -324.90735 EV
          ELECTRONIC ENERGY       =       -474.27017 EV
          CORE-CORE REPULSION     =        149.36282 EV
          GRADIENT NORM           =          0.65369 = 0.37741 PER ATOM
          DIPOLE                  =          1.73858 DEBYE    POINT GROUP:       C2v 
          NO. OF FILLED LEVELS    =          4
          IONIZATION POTENTIAL    =         12.316247 EV
          HOMO LUMO ENERGIES (EV) =        -12.316  4.058
          MOLECULAR WEIGHT        =         18.0152
          COSMO AREA              =         42.43 SQUARE ANGSTROMS
          COSMO VOLUME            =         25.15 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     2    H     1     1.53633
            O     3    H     1     0.56094
            H     1    H     2     0.00000
          SCF CALCULATIONS        =          1
          WALL-CLOCK TIME         =          0.020 SECONDS
          COMPUTATION TIME        =          0.020 SECONDS


          FINAL GEOMETRY OBTAINED
 PM3 1SCF GRAD CHARGE=0 Singlet

 Mopac file generated by Gabedit
  H     0.00000000 +1   0.77870100 +1  -0.44376600 +1
  H     0.00000000 +1  -0.75754000 +1  -0.46039600 +1
  O     0.00000000 +1   0.00472800 +1   0.10883100 +1
 
