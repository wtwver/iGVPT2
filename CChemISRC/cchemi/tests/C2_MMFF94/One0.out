
FINAL ENERGY:    0.00000000000000 kcal/mol
Gradients: 2 atoms 
    -0.00000000000000    -0.00000000000000    -0.00000130769825                   C1
    -0.00000000000000    -0.00000000000000     0.00000130769825                   C1

